DFXswiss/api
# [DEV-728], [DEV-740], [DEV-818], [DEV-892] Fees calculation and persistence, referencing assets, batch optimisation

### Release Checklist

#### Pre-Release

- [x] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [x] Check for linter errors (in PR)
- [x] Test basic user operations
  - Login/logout
  - Registration
  - Update user data
  - Create/delete payment routes
- [x] Make sure Asset table is up to date (incl Token for DFI and native coins for every blockchain)

#### Post-Release

- Test basic user operations
- Monitor application insights log
